src/core/server/decorators/interaction.ts

```diff
@@ -0,0 +1,45 @@
+import alt, { IVector3 } from 'alt-server';
+import { Athena } from '../api/athena';
+
+/**
+ * @param {IVector3} pos - The position of the interaction.
+ * @param {string} description - The description of the interaction.
+ * @param {number} range - The range of the interaction.
+ * @param {number} dim - The dimension to search in.
+ * @param {string} uid - The uid of the interaction.
+ * Decorator for Athena's Interaction Controller
+ * ```
+ * class ExampleDecorators {
```

Comment:
Should be `ExampleInteractions` - decorators implies that you have more than one decorator tbh.